README.md

```diff
@@ -2,12 +2,13 @@
 
 OpenWiFiMap is a database and map for free network WiFi routers (freifunk and others, too!). 
 
-This is the database part of the openwifimap. Make sure to also take a look at the [openwifimap HTML5 app](https://github.com/freifunk/openwifimap-html5) running at [openwifimap.net](http://openwifimap.net).
+This is the database part of the openwifimap. Make sure to also take a look at the [openwifimap HTML5 app](https://github.com/freifunk/openwifimap-html5).
 
 # API installations
 This is a (possibly incomplete) list of API installations:
-* [api.openwifimap.net](http://api.openwifimap.net)
-* [couch.pberg.freifunk.net/test/_design/owm-api/_rewrite](http://couch.pberg.freifunk.net/test/_design/owm-api/_rewrite/)
```

Comment:
At least "couch.pberg." seems really gone.

Reply:
@sarumpaet should I update this PR or will you do on the fly?

Reply:
The Python backend's README doesn't have that section anymore. I don't think a software component's README is a good place to collect such references, and I'd be surprised if there were any installations of the Python backend apart from api.openwifimap.net at the moment... let's have a look again in a year, perhaps. :)

Reply:
OK, your point ...